go/hyperkit.go

```diff
@@ -70,6 +69,12 @@ type Socket9P struct {
 	Tag  string `json:"tag"`
 }
 
+// DiskImage contains the path to a disk image and an optional size if the image needs to be created.
+type DiskImage struct {
+	Path   string `json:"path"`
+	SizeMB int    `json:"size"`
```

Comment:
Can we call this `DiskConfig` and `Size` in the same way as https://github.com/linuxkit/linuxkit/blob/master/src/cmd/linuxkit/util.go#L139
The code is in different repos, but might as well try to keep it consistent